### ISSUE TITLE
reduce code duplication in TwoSpringView

### ISSUE THREAD

**Issue by jbphet** [phetsims/masses-and-springs#109]
I'm seeing a lot of duplicated code in the file `TwoSpringView.js`.  Here is an example:

```js
    // @public Initializes equilibrium line for first spring
    this.firstSpringEquilibriumLineNode = new EquilibriumLineNode(
      modelViewTransform2,
      model.springs[ 0 ],
      model.equilibriumPositionVisibleProperty,
      tandem.createTandem( 'firstSpringEquilibriumLineNode' )
    );

    // @public Initializes equilibrium line for second spring
    this.secondSpringEquilibriumLineNode = new EquilibriumLineNode(
      modelViewTransform2,
      model.springs[ 1 ],
      model.equilibriumPositionVisibleProperty,
      tandem.createTandem( 'secondSpringEquilibriumLineNode' )
    );
```

The bulk of the code is exactly the same with just a couple of adjustments for the different springs.  This pattern occurs several times in the file.  It would be better if the springs were in an array, and there was a loop where the various controls and lines were added based on their positions.

Also, there is a lot of similarity in `OneSpringView`.  It would be best, if possible, for there to be just one file called `SpringView` that took an array of springs and added all the nodes needed for it.

This was observed during code review, see #36.

**Comment by jbphet** [comment]
Here's a thought, and @Denz1994 - you'll have to decide if this is a helpful suggestion.  Maybe some of the common elements, such as the lines, can be moved into `OscillatingSpringNode`.

**Comment by Denz1994** [comment]
A new common view was created for the two/one spring view system. The hierarchy now exists as such:
- SpringView.js/TwoSpringView.js/IntroScreenView.js
- SpringView.js/OneSpringView.js/EnergyScreenView.js

The same hierarchy is followed for the Vector Screens and Lab Screens.

**Comment by Denz1994** [comment]
After some discussion with @jbphet, we went over some objects that could be consolidated into a more common view (SpringView.js). The methods have been added to remove code duplication when generating view objects. Also, an array has bee created for the oscillating spring nodes in the common SpringView.js

Closing this issue because SpringView.js, TwoSpringView.js, and OneSpringView.js have been consolidated.
